packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,682 @@
+---
+title: Factor Profiles
+category: management
+---
+
+# Factor Profiles API
+
+The Okta Factor Profiles API enables an Administrator configure which factor profiles are available to use for multi-factor authentication.
+
+> **Note:** In the Admin Dashboard
+
+The factor profiles API supports the following **profile operations**:
+
+* Get all profiles for a specific factor type
+* Create, read, update, and delete a profile
+
+The factor profiles API supports the following **profile feature operations**:
+
+* Get all features for a profile
+* Update a feature for a profile
+
+## Factor Profile Operations
+
+### Get a Factor Profile
+
+<ApiOperation method="get" url="/api/v1/org/factors/${factorName}/profiles/${profileId}" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+| `profileId`  | String | The factor profile's unique identifier.         |
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}"
+```
+
+##### Response Example
+
+HTTP 200:
+[Factor Profile Object](#factor-profile-object)
+
+### Get All Factor Profiles by Factor Type
+
+<ApiOperation method="get" url="/api/v1/org/factors/${factorName}/profiles" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+
+#### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles"
+```
+
+#### Response Types
+
+HTTP 200:
+[Factor Profile Object](#factor-profile-object)
+
+### Delete a Factor Profile
```

Comment:
Better to order examples of CRUD for profile

Reply:
Not sure what you mean @AjayThomas-okta